src/agent/oci/src/lib.rs

```diff
@@ -733,14 +739,20 @@ pub const ARCHS390: &str = "SCMP_ARCH_S390";
 pub const ARCHS390X: &str = "SCMP_ARCH_S390X";
 pub const ARCHPARISC: &str = "SCMP_ARCH_PARISC";
 pub const ARCHPARISC64: &str = "SCMP_ARCH_PARISC64";
+pub const ARCHRISCV64: &str = "SCMP_ARCH_RISCV64";
+
+pub type LinuxSeccompFlag = String;
 
 pub type LinuxSeccompAction = String;
 
 pub const ACTKILL: &str = "SCMP_ACT_KILL";
+pub const ACTKILLPROCESS: &str = "SCMP_ACT_KILL_PROCESS";
+pub const ACTKILLTHREAD: &str = "SCMP_ACT_KILL_THREAD";
 pub const ACTTRAP: &str = "SCMP_ACT_TRAP";
 pub const ACTERRNO: &str = "SCMP_ACT_ERRNO";
 pub const ACTTRACE: &str = "SCMP_ACT_TRACE";
 pub const ACTALLOW: &str = "SCMP_ACT_ALLOW";
+pub const ACTLOG: &str = "SCMP_ACT_LOG";
```

Comment:
Looks like we could add notify now, but I'm happy adding that in a follow on commit...

Reply:
Thank you for the comment. Yes, we can add the seccomp notify action now, but it was just added to the [OCI spec](https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#seccomp) 3 weeks ago by the [commit](https://github.com/opencontainers/runtime-spec/pull/1074).
So, I could not add the feature at the time of this pull request.
I'll add the action when the kata is ready to support the notify.